runtime/api/WeexInstance.js

```diff
@@ -128,6 +129,31 @@ export default class WeexInstance {
     return null
   }
 
+  importScript (src, options = {}) {
+    const id = getId(this)
+    const taskCenter = getTaskCenter(id)
+    return new Promise((resolve, reject) => {
+      if (!taskCenter || typeof !taskCenter.send === 'function') {
```

Comment:
`typeof !taskCenter.send === 'function' ` should be `typeof taskCenter.send !== 'function'`